### PR TITLE
Add SetPaidCommand

### PR DESCRIPTION
**Pull request by guokweijie**

### CODE REVIEW

**Review by Fuzanwenn — CHANGES_REQUESTED** [review]
Empty Java doc for SetPaidCommand class

**Review by SpecOps2016 — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/SetPaidCommand.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.bill.Bill;
+
+/**
+ *
```
[the remark this answers is not in the document]
Comment (guokweijie) [reply src/main/java/seedu/address/logic/commands/SetPaidCommand.java]: ok thanks!

**Review by Fuzanwenn — APPROVED** [review]
LGTM

**Pull request merged**